packages/datatrak-web/src/api/queries/useSurveys.ts

```diff
@@ -14,7 +14,7 @@ export const useSurveys = (selectedCountryName?: Entity['name'], projectId?: Pro
     (): Promise<DatatrakWebSurveyRequest.ResBody[]> =>
       get('surveys', {
         params: {
-          fields: ['name', 'code', 'id', 'survey_group.name', 'countryNames'],
+          fields: ['name', 'code', 'id', 'survey_group.name'],
```

Comment:
```suggestion
          fields: ['name', 'code', 'id', 'survey_group.name', 'countryNames'],
```
I think this is another merge issue but we should still get the countryNames here

Reply:
Same here